### PR TITLE
Add possibility to exclude dashboard from update

### LINKED ISSUE #14: Add possibility to exclude dashboard from `helm upgrade` process

### LINKED ISSUE #15: Add release version badge to the README

### LINKED ISSUE #16: Add possibility to customize logging format
- [x]  default
- [x] json

### PR DESCRIPTION
**Pull request by matofeder**
closes https://github.com/dNationCloud/kubernetes-jsonnet-translator/issues/16   
closes https://github.com/dNationCloud/kubernetes-jsonnet-translator/issues/15   
closes https://github.com/dNationCloud/kubernetes-jsonnet-translator/issues/14

### CODE REVIEW

File: translator/main.py [hunk 1]
```diff
@@ -306,26 +322,36 @@ def create_rules_object(args_, jsons, user_labels, user_annotations):
...
         user_labels (dict):  Labels of kubernetes object.
         user_annotations (dict): Annotations of kubernetes object.
+        translator_annotations (dict): Translator specific annotations.
 
     Returns:
         None
     """
     v1 = client.CoreV1Api()
 
+    dashboards_skip = translator_annotations.get("grafana_dashboards_skip_update", [])
```
Comment (davidsuba) [inline translator/main.py:344]: all translator annotations (like `grafana_label`) have possibility to be configured via arguments of translator (except `jsonnet_filename` but that could be changed too)

File: translator/main.py [hunk 2]
```diff
@@ -36,6 +37,27 @@
...
+    "max_stack",
+    "gc_min_objects",
+    "gc_growth_trigger",
+    "ext_vars",
+    "ext_codes",
+    "tla_vars",
+    "tla_codes",
+    "max_trace",
+]
+SUPPORTED_TRANSLATOR_ARGS = [
```
Comment (davidsuba) [inline translator/main.py:50]: maybe other translator arguments can be added here, like `grafana_label`, `prometheus_label` and `jsonnet_filename`

File: translator/main.py (hunk 1 above)
Comment (matofeder) [reply translator/main.py:344]: Addressed 8a6a5613aed28b283875ff5dfce6e1a30fa881d6

File: translator/main.py (hunk 2 above)
Comment (matofeder) [reply translator/main.py:50]: Addressed 8a6a5613aed28b283875ff5dfce6e1a30fa881d6

**Review by davidsuba — APPROVED** [review]

**Pull request merged**